src/target/adiv5.c

```diff
@@ -573,7 +581,7 @@ static void adiv5_component_probe(
 		DEBUG_INFO("%sROM: Table END\n", indent);
 
 	} else {
-		if (designer_code != JEP106_MANUFACTURER_ARM) {
+		if ((designer_code != JEP106_MANUFACTURER_ARM) && (designer_code != JEP106_MANUFACTURER_ARM_CHINA)) {
```

Comment:
Drop the extra parens, they aren't necessary and suppress a GCC warning about accidental assignment that we want.

Reply:
ok, done.